PovioKit.podspec

```diff
@@ -62,4 +62,13 @@ Pod::Spec.new do |s|
       cs.source_files = 'PovioKit/Classes/Views/GradientView/**/*'
     end
   end
+  
+  s.subspec 'Networking' do |sp|
+    sp.source_files = 'PovioKit/Classes/Networking/**/*'
+    
+    sp.subspec 'RestClient' do |cs|
+      cs.source_files = 'PovioKit/Classes/Networking/RestClient/**/*'
+      cs.dependency 'Alamofire'
```

Comment:
I don't like this dependency in our pods. What do you think about switching to url session instead?

Reply:
Why not? We've been using Alamofire in most of our projects thus far...

Reply:
What I propose instead is that we make `RestClient` as general as possible, meaning that we also abstract away the underlying HTTP request engine. 
We could then implement two (or more) such interfaces: `URLSession` implementation and `Alamofire` implementation. Users of the lib could then decide which implementation they want: `pod 'PovioKit/Networking/RestClient/Alamofire' or 
`pod 'PovioKit/Networking/RestClient/URLSession'
What do you think?

Reply:
Yes, Alamofire should be opt-in. Let's proceed.